model/project.go

```diff
@@ -1202,6 +1202,10 @@ func FindLatestVersionWithValidProject(projectId string) (*Version, *Project, er
 		}))
 	}
 
+	if lastGoodVersion == nil {
+		return nil, nil, errors.WithStack(errors.Errorf("could not find a valid version for project %s", projectId))
```

Comment:
We can still use the error that's wrapped at 1209, just not with lastGoodVersion.Identifier, so either we could change that error statement to not use it or have this error be different but still an errors.Wrapf()

Reply:
done

Reply:
done